testdata/serviceconfig.yml

```diff
@@ -7,6 +7,23 @@ rules:
   aspects:
   - kind: quotas
     params:
+  - kind: metrics
+    adapter: "prometheus"
+    params:
+      metrics:
+      - descriptor: "request_count"
+        value: "1"
```

Comment:
Turns out it won't actually parse if we use a literal `1`, keeping it as is (`"1"`).